portfolio/src/main/webapp/style.css

```diff
@@ -8,11 +8,23 @@
   margin-top: 20px;
 }
 
-h1 {text-align: center; font-family: Georgia, serif;}
-h3 {font-family: Georgia, serif;}
-h4 {text-align: center; font-family: Verdana, serif;}
+h1 {
```

Comment:
Nit: The changes in this file look mostly like formatting changes (i.e. not to do with adding upload functionality). In the future, it probably would be better to have this in a separate PR though.

Reply:
That's a good advice. I will separate these modifications next time.